installers/mac/tar/templates/Info.plist.template

```diff
@@ -17,11 +17,11 @@
         <key>CFBundlePackageType</key>
         <string>BNDL</string>
         <key>CFBundleShortVersionString</key>
-        <string>1.0</string>
```

Comment:
I cannot confirm if there's any side effect here. I prefer to keep it as `1.0` as other JDK distributions do.

Reply:
There should not be any side affect here, however, as you mentioned, that can't really be confirmed. I have a feeling this value has just been overlooked for so long that it's now just frozen at `1.0`. I'll see if I can bring it up to the upstream project.